worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +69,43 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// Initialise lives
+	int lives = 4;
+
+	// Initialise guessed word
+	std::string guess;
+
+	// Game loop
+	while (lives > 0) {
+
+		// Promt player to guess
+		std::cout << std::endl << "You have " << lives << " chances to guess the word, pick your keystrokes wisely." << std::endl << std::endl;
+
+		std::cin >> guess;
+
+		// Applies check to see if word is in list
+		if (wordInList(guess, options)) {
+
+			lives -= 1;
+
+			// Check to see if word is correct
+			if (guess == secret) {
+				std::cout << "Congrats, you're a real winner. Go upstairs and tell your mum, she will be proud." << std::endl;
+				lives = 0;
```

Comment:
Needs a comment saying that this breaks out of the loop. Or better, just use the `break` keyword